src/stylesheets/global.css

```diff
@@ -111,6 +111,7 @@ iframe,
 img,
 svg,
 video {
+  max-width: 100%;
```

Comment:
Oh, I forgot that I should probably briefly explain this. In https://github.com/ShelterTechSF/sheltertech.org/pull/370, I got rid of the legacy style sheets, but had copy over a few of the reset styles manually over to our new site's stylesheet. This one was actually originally in the old reset styles, but I had missed copying it over because it didn't look like any of the images at the time needed it. Since I'm trying to add images at 2x resolution so that they look nicer on high density displays, like Apple Retina displays, we really need this so that images don't grow wider than their parent element.